docs/src/install/gettingstarted.rst

```diff
@@ -0,0 +1,149 @@
+
+You can fine-tune the distribution and algorithm with many options either with more arguments or by
+using a configuration file. Learn more at `/user/api`.
+
+Scaling up
+----------
+
+Due to the native asynchronous nature of Oríon, it's very easy to run it on parallel or distributed
+environments. Oríon does not rely on a traditional master/slave setup. The synchronization point is
+the database, each worker will separately generate a new trial based on the state of the experiment
+stored in the database.
+
+Make sure to visit :doc:`/user/parallel` to learn more about it and checkout the tutorial to run
+Oríon in :doc:`HPC environments </tutorials/cluster>`.
+
+Search Space
+============
+
+The search space is defined by priors for each hyperparameter to optimize. In the snippet earlier,
+we used the *loguniform* prior. We support almost all the distributions from `scipy
+<https://docs.scipy.org/doc/scipy/reference/stats.html>`_ out of the box. You can define them either
+directly in the command line (as shown previously) or in a configuration file:
+
+.. code-block:: yaml
+
+    lr: 'orion~loguniform(1e-5, 1.0)'
+
+And then use it with:
+
+.. code-block:: console
+
+   $ orion hunt -n <experiment name> script.py --config config.yaml
+
+Make sure to visit :doc:`/user/searchspace` for an exhaustive list of priors and their parameters.
+
+Algorithms
+==========
+
+Similarly to search spaces, Oríon supports multiple algorithms out of the box: :ref:`random-search`,
+:ref:`ASHA`, and :ref:`hyperband-algorithm`. Each one is fully configurable through the
+configuration file. The samples of hyperparameter are based on the previous trials.
+
+Make sure to checkout `this presentation
+<https://docs.google.com/presentation/d/18g7Q4xRuhMtcVbwmFwDfH7v9gKS252-laOi9HrEQ7a4/present?slide=id.g6ba6d709b9_4_19>`_
+for an quick overview of each algorithm and to visit :doc:`/user/algorithms` to learn about the
+algorithms and get recommendations about their use cases.
+
+Monitoring
+==========
+
+Oríon offers different ways to get information about your experiments and trials.
+
+* ``$ orion list`` gives an overview of all the experiments.
+* ``$ orion status`` gives an overview of trials for experiments.
+* ``$ orion info`` gives a detailed description of a given experiment such as priors and best
+  trials.
+
+Each command is described in detail in :doc:`/user/monitoring`.
+
+If you want a more fine grained approach, you can always query the database directly or via Oríon's
+python API. Check out :doc:`/user/storage` for more information.
+
+Next steps
+==========
+
+It's worth to take a look at the :doc:`configuration system </user/config>` to learn more about how
+to make the most out of Oríon and define precise behaviors for your algorithms and experiments.
+
+Explore the :doc:`User Manual </user/overview>`, oríon is simple from the outside but is feature
+rich! We also have a few tutorials available. If you're a researcher or develop you might be
```

Comment:
Maybe turn the word tutorial into a link pointing to simple example?

Reply:
I wanted to add a list of tutorials at first but we don't have such thing yet so instead, I'm adding a few examples.